.editorconfig

```diff
@@ -1,5 +1,20 @@
 # EditorConfig is awesome: http://EditorConfig.org
 
+# Create portable, custom editor settings with EditorConfig
```

Comment:
This file is copied from other repos. It contains the rules collected over the years of work in OSS.